starport/services/chain/plugin-launchpad.go

```diff
@@ -16,11 +18,20 @@ import (
 
 type launchpadPlugin struct {
 	app App
+	cmd chaincmd.ChainCmd
 }
 
 func newLaunchpadPlugin(app App) *launchpadPlugin {
+	// initialize the chain command with keyring backend test
+	cmd := chaincmd.New(
+		app.D(),
+		chaincmd.WithKeyrinBackend("test"),
```

Comment:
It would be nice to create a type and constants that uses this type to declare each keyring backend, so we can make `WithKeyrinBackend` to accept that type instead.